README.md

```diff
@@ -82,21 +82,41 @@ Upon completion of the secure training (or inference) job, the models (or predic
 
 For more instructions, please refer to [mpc examples](./python/paddle_fl/mpc/examples)
 
+
 ## Installation
 
+We provide three ways to install PaddleFL：
+
+1. Use PaddleFL in docker
+
 We **highly recommend** to run PaddleFL in Docker
 
 ```sh
 #Pull and run the docker
 docker pull paddlepaddle/paddlefl:1.1.2
 docker run --name <docker_name> --net=host -it -v $PWD:/paddle <image id> /bin/bash
 
-#Install paddle_fl
+```
+
+2. Install PaddleFL via installation package
+
+We provide compiled PaddlePaddle and PaddleFL installation packages, you can download and install them directly.
+```
+#Install PaddlePaddle
+wget https://paddlefl.bj.bcebos.com/paddlepaddle-1.8.5-cp**-cp**-linux_x86_64.whl
+pip3 install paddlepaddle-1.8.5-cp**-cp**-linux_x86_64.whl
+(Replace ** with the python version in the installation environment, for example, python3.8 corresponds to cp38)
```

Comment:
you can give a detailed example, like:

E.g., if you are using python3.6, the commands are as below:
'''wget https://paddlefl.bj.bcebos.com/paddlepaddle-1.8.5-cp3.6-cp3.6-linux_x86_64.whl
pip3 install paddlepaddle-1.8.5-cp3.6-cp3.6-linux_x86_64.whl'''